x-pack/plugins/security_solution/public/common/components/navigation/use_security_solution_navigation/use_primary_navigation.tsx

```diff
@@ -26,7 +26,7 @@ export const usePrimaryNavigation = ({
 }: PrimaryNavigationProps): KibanaPageTemplateProps['solutionNav'] => {
   const isGroupedNavigationEnabled = useIsGroupedNavigationEnabled();
   const mapLocationToTab = useCallback(
-    (): string => ((tabName && navTabs[tabName]) || navTabs[pageName])?.id ?? '',
+    (): string => (navTabs[pageName] || (tabName && navTabs[tabName]))?.id ?? '',
```

Comment:
@semd We had spoken about this and I can't quite remember the conclusion, was it that we can fully remove the condition on the right? Or was it that a certain team didnt use `pageName` or something like that? 🤔

Reply:
Hey KP, yes all the pages under _/administration/*_ use the `tabName` here, despite they are not actual tabs. We have to remove the `navTabs.administration` to make the old navigation highlight the correct pages. I don't think we need this entry since there's no main "administration" page in the old navigation, but we'll have to test it extensively to make sure that removing it does not cause any side effects.
A similar pattern has been implemented with new CSP pages, they are all under _/cloud_security_posture/*_ we'll have to check those as well.

Reply:
Thanks @semd!

Tested to ensure all side nav tab-highlighting functionality remains intact. This required the removal of `SecurityPageName.administration`.